website_form_recaptcha/data/ir_config_parameter_data.xml

```diff
@@ -6,18 +6,15 @@
 -->
 
 
-<openerp>
+<odoo>
     <data noupdate="1">
```

Comment:
Remove data tag and put noupdate directly in odoo tag please

Reply:
:+1: